packages/dds/sequence/src/test/sharedString.spec.ts

```diff
@@ -648,6 +657,95 @@ describe("SharedString", () => {
 				"Could not annotate marker in remote string",
 			);
 		});
+
+		// TODO: these three tests currently fail with 0x2d8
+		// AB#19722
+		it.skip("zero length obliterate at the start of the string", () => {
+			sharedString.insertText(0, "0123456789");
+			containerRuntimeFactory.processAllMessages();
+			assert.equal(
+				sharedString.getText(),
+				sharedString2.getText(),
+				"starting state should be equal",
+			);
+
+			sharedString.obliterateRange(
+				{ pos: 4, side: Side.After },
+				{ pos: 5, side: Side.Before },
+			);
+			sharedString.insertText(5, "AAA");
+			sharedString2.obliterateRange(
+				{ pos: 4, side: Side.After },
+				{ pos: 5, side: Side.Before },
+			);
+			sharedString2.insertText(5, "BBB");
+
+			containerRuntimeFactory.processAllMessages();
+			assert.equal(
+				sharedString.getText(),
+				sharedString2.getText(),
+				"end state should be equal after obliterate",
+			);
+			assert.equal(sharedString2.getText(), "01AAA2456789", "obliterate failed");
+		});
+
+		it.skip("zero length obliterate in the middle of the string", () => {
```

Comment:
This should be the only test. start and end are not supported, and the other tests don't actually test start and end anyway.